src/mscorlib/corefx/System/Globalization/JapaneseCalendar.Windows.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics;
+
```

Comment:
The *.Windows.cs files are not getting compiled in CoreCLR. They are in CoreCLR just for reference.